extension/src/setup/index.ts

```diff
@@ -310,6 +311,9 @@ export class Setup
     const root = getFirstWorkspaceFolder()
     if (root) {
       await this.dvcExecutor.init(root)
+      void Toast.infoWithOptions(
```

Comment:
Another option that came to mind was adding an action in the notification that runs `git commit` for the user.

Reply:
Would it be bad to simply commit for the user while running `dvc init`? Like simply adding the necessary files and commit with a message saying `dvc init`. What does everyone think @iterative/vs-code ?

Reply:
I think we are trying to mitigate `exp show` & `exp run` from breaking if there are **no commits** in the repo. 

The DVC team is actively working on the other side of this (https://github.com/iterative/dvc/pull/8840)
I also don't think a Toast will be enough by itself. Generally, I'd agree that we could just commit automatically for the user. However, there must be some rationale behind `dvc init` not auto-committing everything (perhaps if there is a dirty workspace or the user might want to commit different files together). An extra step in Setup is probably warranted but only if there are no commits in the repo.

Reply:
Agreed! I'll go ahead and merge this for a first iteration, adding a setup step in the followup, since it will take me a bit of time to add a Setup step (Looks like I'll need to add a run a git command and check the error message to check for 0 git commits 🤔).